_api-reference/document-apis/update-by-query.md

```diff
@@ -56,15 +56,18 @@ search_type | String | Whether OpenSearch should use global term and document fr
 search_timeout | Time | How long to wait until OpenSearch deems the request timed out. Default is no timeout.
 slices | String or integer | The number slices to split an operation into for faster processing, specified by integer. When set to `auto` OpenSearch it should decides how many the number of slices for the operation. Default is `1`, which indicates an operation will not be split.
 sort | List | A comma-separated list of &lt;field&gt; : &lt;direction&gt; pairs to sort by.
-_source | String | Whether to include the `_source` field in the response.
-_source_excludes | String | A comma-separated list of source fields to exclude from the response.
-_source_includes | String | A comma-separated list of source fields to include in the response.
+_source | String | Controls the parts of the document source available to the script and returned in the response. Excluded fields may be removed from the document if not reassigned in the script. 
```

Comment:
`parts of the document`: possible to make this more clear? I find the word `parts` ambiguous. Like fields of the document source available?

Excluded fields may be removed from the document **_source** if not reassigned in the script?